engine/src/main/java/org/terasology/engine/modes/loadProcesses/InitialisePhysics.java

```diff
@@ -16,11 +16,10 @@
 package org.terasology.engine.modes.loadProcesses;
 
 import org.terasology.context.Context;
+import org.terasology.engine.modes.SingleStepLoadProcess;
```

Comment:
Why was there an import missing? Is it because the file was moved to a different package?

Reply:
yep